src/fmu/dataio/preprocessed.py

```diff
@@ -186,9 +187,7 @@ def _get_updated_metadata(self, meta_existing: dict, objfile: Path) -> dict:
         try:
             # TODO: Would like to use meta.Root.model_validate() here
             # but then the '$schema' field is dropped from the meta_existing
-            validated_metadata = schema.InternalObjectMetadata.model_validate(
-                meta_existing
-            )
+            validated_metadata = ObjectMetadataExport.model_validate(meta_existing)
```

Comment:
If we could add the `$schema` to the `Root` model instead, we could address the TODO here